pkg/sbom/implementation.go

```diff
@@ -350,7 +352,7 @@ func buildDocumentSPDX(spec *Spec, doc *bom) (*spdx.Document, error) {
 	}
 
 	spdxDoc := spdx.Document{
-		ID:      fmt.Sprintf("SPDXRef-DOCUMENT-%s", fmt.Sprintf("apk-%s-%s", spec.PackageName, spec.PackageVersion)),
+		ID:      "SPDXRef-DOCUMENT",
```

Comment:
While we are at this, I recommend fixing the namespace (line 367) to make it unique to the package and deterministic. You can hash the previous value here to for it, eg:

```
https://spdx.org/spdxdocs/chainguard/melange/668e3da6b9af6045a9...,
```